File: app/src/main/java/com/zwstudio/logicpuzzlesandroid/puzzles/fourmenot/domain/FourMeNotTreeObject.java
package com.zwstudio.logicpuzzlesandroid.puzzles.fourmenot.domain;

import com.zwstudio.logicpuzzlesandroid.common.domain.AllowedObjectState;

public class FourMeNotTreeObject extends FourMeNotObject {
    public AllowedObjectState state = AllowedObjectState.Normal;
    public String objAsString() {
        return "tree";
    }
}

File: app/src/main/java/com/zwstudio/logicpuzzlesandroid/puzzles/fourmenot/domain/FourMeNotTreeObject.kt
package com.zwstudio.logicpuzzlesandroid.puzzles.fourmenot.domain

import com.zwstudio.logicpuzzlesandroid.common.domain.AllowedObjectState

class FourMeNotTreeObject : FourMeNotObject() {
    var state = AllowedObjectState.Normal
    override fun objAsString(): String {
        return "tree"
    }
}